nifi-nar-bundles/nifi-workday-bundle/nifi-workday-processors/src/main/java/org/apache/nifi/processors/workday/GetWorkdayReport.java

```diff
@@ -0,0 +1,442 @@
+        .build();
+
+    protected static final PropertyDescriptor WEB_CLIENT_SERVICE = new PropertyDescriptor.Builder()
+        .name("Web Client Service Provider")
+        .description("Web client which is used to communicate with the Workday API.")
+        .required(true)
+        .identifiesControllerService(WebClientServiceProvider.class)
+        .build();
+
+    protected static final PropertyDescriptor FIELDS_TO_HASH = new PropertyDescriptor.Builder()
+        .name("Hashed Fields")
+        .displayName("Hashed Fields")
+        .description("Comma separated record paths to be replaced with their corresponding hash.")
+        .required(false)
+        .expressionLanguageSupported(FLOWFILE_ATTRIBUTES)
+        .addValidator(NON_BLANK_VALIDATOR)
+        .build();
+
+    protected static final PropertyDescriptor HASHING_ALGORITHM = new PropertyDescriptor.Builder()
+        .name("Hashing Algorithm")
+        .displayName("Hashing Algorithm")
+        .description("Determines what hashing algorithm should be used to perform the hashing function.")
+        .required(true)
+        .allowableValues(HashAlgorithm.SHA256.getName(), HashAlgorithm.SHA512.getName())
+        .defaultValue(HashAlgorithm.SHA256.getName())
+        .addValidator(NON_BLANK_VALIDATOR)
+        .expressionLanguageSupported(NONE)
+        .dependsOn(FIELDS_TO_HASH)
+        .build();
+
+    protected static final PropertyDescriptor RECORD_READER_FACTORY = new PropertyDescriptor.Builder()
+        .name("record-reader")
+        .displayName("Record Reader")
+        .description("Specifies the Controller Service to use for parsing incoming data and determining the data's schema.")
+        .identifiesControllerService(RecordReaderFactory.class)
+        .required(false)
+        .build();
+
+    protected static final PropertyDescriptor RECORD_WRITER_FACTORY = new PropertyDescriptor.Builder()
+        .name("record-writer")
+        .displayName("Record Writer")
+        .description("The Record Writer to use for serializing Records to an output FlowFile.")
+        .identifiesControllerService(RecordSetWriterFactory.class)
+        .dependsOn(RECORD_READER_FACTORY)
+        .required(true)
+        .build();
+
+    protected static final Relationship ORIGINAL = new Relationship.Builder()
+        .name("Original")
+        .description("Request FlowFiles transferred when receiving HTTP responses with a status code between 200 and 299.")
+        .build();
+
+    protected static final Relationship FAILURE = new Relationship.Builder()
+        .name("Failure")
+        .description("Request FlowFiles transferred when receiving socket communication errors.")
+        .build();
+
+    protected static final Relationship RESPONSE = new Relationship.Builder()
+        .name("Response")
+        .description("Response FlowFiles transferred when receiving HTTP responses with a status code between 200 and 299.")
+        .build();
```

Comment:
Although not completely consistent, the general pattern for relationships is all lowercase. `InvokeHTTP` is an exception to this pattern, and although these names are similar, recommend renaming to `original`, `failure`, and `success`.

Reply:
done